README.md

```diff
@@ -39,6 +39,34 @@ Using the Datadog Operator requires the following prerequisites:
 
 See the [Quick Start tutorial](docs/quick-start.md).
 
+## Metrics and Events
+
+The Datadog Operator sends metrics and events to Datadog to monitor the Datadog Agent components deployment in the cluster.
+
+### Metrics
+
+|metric name   |metric type   |description   |
+|---|---|---|
+|datadog.operator.agent.deployment.success   |gauge   |1 if the de desired number of Agent replicas equals the number of available Agent pods, 0 otherwise   |
+|datadog.operator.clusteragent.deployment.success   |gauge   |1 if the de desired number of Cluster Agent replicas equals the number of available Cluster Agent pods, 0 otherwise   |
```

Comment:
```suggestion
| `datadog.operator.clusteragent.deployment.success` | gauge   | `1` if the de desired number of Cluster Agent replicas equals the number of available Cluster Agent pods, `0` otherwise.   |
```